### DISCUSSION TITLE
Changing Default View for Particular Model?

### DISCUSSION THREAD

**Discussion by adparis99** [netbox-community/netbox#12291] [Q&A]
This may be a stupid question with an obvious answer of no, but figured I'd ask anyway. We utilize IPAM quite a bit, particularly prefixes and IP addresses. Almost every time I open the main page for a prefix (https://10.1.34.4/ipam/prefixes/{id}), I am looking for the IP Addresses sub-page (https://10.1.34.4/ipam/prefixes/{id}/ip-addresses/), since it provides excellent visibility for what is in-use, available, etc. Is there any support within NetBox to customize behavior such that the IP Address page could be set as the default view for prefixes? Or is this a pipe dream with a limited use case?

**Comment by dreng** [comment]
Short answer: No, this is not possible because the paths are hard-coded.

If you REALLY want that you may create a plugin where you are able open the desired path from the main menu directly. But in my opinion that's completely overkill just for saving two clicks.
